scene/3d/collision_object_3d.cpp

```diff
@@ -621,6 +621,8 @@ void CollisionObject3D::shape_owner_add_shape(uint32_t p_owner, const Ref<Shape3
 	total_subshapes++;
 
 	_update_shape_data(p_owner);
+
+	update_gizmos();
```

Comment:
```suggestion
	_update_shape_data(p_owner);
	update_gizmos();
```

Reply:
Nitpick, but worth changing for consistency with other such "grouped" updates in other classes. CC @jsjtxietian

Reply:
Yes absolutely, thanks !